testing_and_setup/compass/general.config.ocean

```diff
@@ -9,23 +9,23 @@
 # init namelists in the default_inputs directory after a successful build of
 # the ocean model.
 [namelists]
```

Comment:
Please remove this file from your PR.  What I often do is have this in its own commit and use interactive rebase to keep it at the head, then when I push do something like this, where key thing is the `HEAD^` that removes this before you push:

`git push remote HEAD^:remote/branch/name`